strongbox-security-api/src/main/java/org/carlspring/strongbox/security/jaas/Roles.java

```diff
@@ -38,4 +38,12 @@ public void setRoles(Set<Role> roles)
         this.roles = roles;
     }
 
+    @Override
+    public String toString()
+    {
+        final StringBuilder sb = new StringBuilder("\n\tRoles{");
+        sb.append("roles=").append(roles);
+        sb.append('}');
+        return sb.toString();
```

Comment:
This rule (empty line before return statement) is missing in XML template that I was imported into IDE and need to be implemented manually (that could be annoing).

Could you modify code style template as well? 

PS I will push that empty line for that case.